### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by wug-ge** [socketio/socket.io#4614] [Q&A] [answered]
Hey,

What's up with the Slack channel?

I'd like to be part of the community and chat about Socket.io, the Link on the Websites gives me an "Application Error".
https://socketio-slackin.herokuapp.com/

If Slack is not used anymore, maybe remove it from the website...

**Comment by darrachequesne** [comment, marked as answer]
Hi! There was an issue with the Slack integration, it should be working now: https://socketio-slackin.herokuapp.com/

Sorry for the delay!

**Reply by moltar** [reply]
It's not working :)

**Comment by vuspenskiy** [comment]
@darrachequesne Heroku: There's nothing here, yet.
